lib/id-generator/pom.xml

```diff
@@ -98,7 +98,7 @@
 	</dependencies>
 
 	<properties>
```

Comment:
we can remove the properties block

Reply:
Double check for unused dependencies (since we removed the transactional context)